pkg/gotool/module_test.go

```diff
@@ -20,10 +19,36 @@ var ModuleStr = `{` +
 	`"Dir":"/home/user/path"}`
 
 var Modules = []gotool.Module{
-	gotool.Module{
-		Path:    "github.com/spf13/cobra",
-		Version: "v0.0.5",
-		Dir:     "/home/user/go/pkg/mod/github.com/spf13/cobra@v0.0.5",
+	{
+		Path:     "bky.sh/pallets/client",
+		Main:     true,
+		Indirect: false,
+		Dir:      "/home/user/Work/bky.sh/pallets/client",
+	},
+	{
+		Path:    "bky.sh/pallets/nub",
+		Version: "v0.0.0",
+		Replace: &gotool.Module{
+			Path: "../nub",
+			Dir:  "/home/user/Work/bky.sh/pallets/nub",
+		},
+		Main:     false,
+		Indirect: false,
+		Dir:      "/home/user/Work/bky.sh/pallets/nub",
+	},
+	{
+		Path:     "cirello.io/pglock",
+		Version:  "v1.8.1-0.20200922151210-b76be34db4ac",
+		Main:     false,
+		Indirect: true,
+		Dir:      "/home/user/go/pkg/mod/cirello.io/pglock@v1.8.1-0.20200922151210-b76be34db4ac",
+	},
+	{
+		Path:     "github.com/spf13/cobra",
+		Version:  "v0.0.5",
+		Main:     false,
+		Indirect: false,
+		Dir:      "/home/user/go/pkg/mod/github.com/spf13/cobra@v0.0.5",
```

Comment:
This testdata was previously split between being declared here in-file and elsewhere in a json file. It's not that big and I think reads easier if kept together so I moved here and deleted the test files